students/lisa_ferrier/lesson02/grid_printer.py

```diff
@@ -0,0 +1,54 @@
+# grid_printer.py
+# Lisa Ferrier, Python 201, Lesson 02
+
+def print_gridV1():
```

Comment:
All function names should be snake_case: def print_grid_v1(): for example.  For this one as well as the others in this code.